go/vt/schema/tablegc.go

```diff
@@ -52,7 +52,7 @@ func (s TableGCState) TableHint() InternalTableHint {
 }
 
 const (
-	OldGCTableNameExpression string = `^_vt_(HOLD|PURGE|EVAC|DROP)_([0-f]{32})_([0-9]{14})$`
+	OldGCTableNameExpression string = `^_vt_(HOLD|PURGE|EVAC|DROP|hold|purge|evac|drop)_([0-f]{32})_([0-9]{14})$`
```

Comment:
This old legacy format is only used in versions before v20. So in newer versions, we don't use this format anymore. Given the next version is already going to be v23, I don't think it's very useful to change this at this point? Since if you're on a new enough version, these old names are not generated anymore? 

In fact, wouldn't it be time to remove the old name support entirely at this point? What do you think @shlomi-noach?

Reply:
Agreed. Tracking issue: https://github.com/vitessio/vitess/issues/14582. Old names could have been deprecated 2 versions ago, but I left it untouched for a while. We should clean up the code now and eradicate any support of old names.
@yushuqin unfortunately for you this means we won't be accepting the PR, apologies.